core/tests/test_trezor.crypto.aesgcm.py

```diff
@@ -61,6 +61,27 @@ def test_gcm(self):
             self.assertEqual(ctx.decrypt(ct), pt)
             self.assertEqual(ctx.finish(), tag)
 
+    def test_gcm_in_place(self):
+        for vector in self.vectors:
+            key, iv, pt, aad, ct, tag = map(unhexlify, vector)
+            buffer = bytearray(pt)
+
+            # Test encryption.
+            ctx = aesgcm(key, iv)
+            if aad:
+                ctx.auth(aad)
+            returned = ctx.encrypt_in_place(buffer)
+            self.assertEqual(buffer, ct)
+            self.assertEqual(returned, len(buffer))
```

Comment:
I would check the tag as well. Here and in three other places below.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/3871/commits/5618f1a06c5c5c4a319d691960f597e19d5a82b5.